backend/internal/adapters/secondary/repositories/querybuilder.go

```diff
@@ -2,6 +2,7 @@ package repositories
 
 import (
 	"github.com/doug-martin/goqu/v9"
+	_ "github.com/doug-martin/goqu/v9/dialect/postgres"
```

Comment:
Is this import needed?

Reply:
Yep! 
![image](https://user-images.githubusercontent.com/24363622/134312525-9fbd36f6-794b-4dac-b8f3-dd2e3d37f2df.png)